src/components/Search/search-result.jsx

```diff
@@ -0,0 +1,96 @@
+import { Link } from 'gatsby';
+import { default as React } from 'react';
+import { Heading } from '@smallstep/step-ui';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  connectStateResults,
+  connectHits,
+  Highlight,
+  Index,
+  Snippet,
+} from 'react-instantsearch-dom';
+import { Box, Typography, ListItem, ListItemText } from '@material-ui/core';
+
+const useStyles = makeStyles((classes) => ({
+  resultsBorder: {
+    borderBottom: `1px solid ${classes.palette.accent.grey3}`,
+  },
+  link: {
+    textDecoration: 'none',
+    color: 'inherit',
+  },
+  hits: {
+    '& mark ': {
+      backgroundColor: classes.palette.accent.yellow,
+    },
+  },
+}));
+
+const HitCount = connectStateResults(({ searchResults }) => {
+  const hitCount = searchResults && searchResults.nbHits;
+  const classes = useStyles();
+  return hitCount > 0 ? (
+    <Box textAlign={'right'} className={classes.resultsBorder} mb={2} mr={1}>
+      <Typography variant="body1">
```

Comment:
Smaller size is good here.

```suggestion
      <Typography variant="body2">
```